.github/workflows/unit-test.yml

```diff
@@ -54,6 +54,52 @@ jobs:
       - name: Report Coverage
         run: npm run codecov
         if: ${{ matrix.node_version == '14' }}
+  node-windows-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Only run the latest LTS version to verify the test can be run on windows.
+        node_version:
+          - "16"
```

Comment:
Why use matrix with only single branch?

Reply:
i guess because other node build should behave the same ? i don't think we need them all

Reply:
Yeah, it's a copy-paste from the node-unit-test. I'll remove it.